scripts/coccinelle/bool_function_type.cocci

```diff
@@ -0,0 +1,19 @@
+@@
```

Comment:
Is this derived from https://github.com/torvalds/linux/blob/master/scripts/coccinelle/misc/boolreturn.cocci?